crates/ide/src/hover/tests.rs

```diff
@@ -1456,9 +1652,9 @@ impl Thing {
     );
     check(
         r#"
-enum Thing { A }
+struct Thing { x: u32 }
 impl Thing {
-    pub fn new() -> Self$0 { Thing::A }
```

Comment:
why was this touched?

Reply:
I didn't change it, just added a check before it for coverage (so the original `check` was moved below, see the check after it). 
But probably the diff algorithm of GitHub doesn't think so, so it incorrectly identifies this line as a modification rather than an insertion.. 🫤